### PR TITLE
Test my progress

### PR DESCRIPTION
**Pull request by DiamantisM**
Test my update please

### CODE REVIEW

File: src/com/Messaris/Scanner.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.Messaris;
+
+public class Scanner {
+
+    public int UserInputNumber() {
```
Comment (MikeFot) [inline src/com/Messaris/Scanner.java:5]: Methods start with lower case letters e.g. `getUserInput()`.

File: src/com/Messaris/Scanner.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.Messaris;
+
+public class Scanner {
```
Comment (MikeFot) [inline src/com/Messaris/Scanner.java:3]: This class should not be named `Scanner` as `Scanner` is a Java Class already and this is confusing.

File: src/com/Messaris/IdontKnowHowToNameThisClass.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.Messaris;
+
+public class IdontKnowHowToNameThisClass {
```
Comment (MikeFot) [inline src/com/Messaris/IdontKnowHowToNameThisClass.java:3]: Yes, you do not know how to name it as the class is architecturally incorrect.

1. You are asking for input as a field and not on demand. Input should be asked for whenever this class is told to (method call), not when the class is created.
2. The random number is generated as part of a field, not when requested for.
3. `z` is assigned as a field, but it is actually a local variable that should only exist inside the method block.
4. You are doing a lot of `new`. `QuoteGenerator`, `NumberFactory` and `Scanner` should be field variables instead e.g.
`QuoteGenerator quoteGenerator = new QuoteGenerator();`
`String quote = quoteGenerator.getOmg();`

**Comment by DiamantisM** [comment]
ok i am making some tweaks

**Pull request closed without merging**